end-user-interviews/README.md

```diff
@@ -0,0 +1,35 @@
+# <img src="https://opentelemetry.io/img/logos/opentelemetry-logo-nav.png" alt="OpenTelemetry Icon" width="45" height=""> OpenTelemetry community End User interview FAQ
+
+## What are end user interviews? 
+
+- The OpenTelemetry community conducts end user interviews to gather high fidelity information from users though the medium of synchronous discussions to learn about their challenges, preferences, and behaviors regarding OpenTelemetry. 
+
+## Why do these interviews happen? 
+
+- Many vendors in this space conduct interviews with their customers but that feedback doesn't always make its way back to the community directly. These interviews will help us test our assumptions, categorize end-users, and describe their needs so that the community can react to their feedback. 
```

Comment:
```suggestion
- These interviews will help us test our assumptions, categorize end-users, and describe their needs so that the community can react to their feedback. 
```